packages/contracts/contracts/Registry.sol

```diff
@@ -28,7 +28,9 @@ contract Registry {
   /// @notice Deploy a contract and add a mapping from counterfactual address to deployed address to storage
   /// @param initcode Contract bytecode concatenated with ABI encoded constructor arguments
   /// @param salt Unique salt used in address computation that allows multiple contracts with the same initcode to exist.
-  function deploy(bytes initcode, uint256 salt) public {
```

Comment:
I think we should only break up into multiple lines like this if we go over a line count. This just makes it harder to read when the line is short.

Reply:
Hmm, wouldn't the linter have fixed it if that were true?

Reply:
That just means the linter isn't configured to fix this.

Reply:
It does split up other functions  across multiple lines though. I think it's just a matter of threshold.